src/screens/Settings/Accounts/index.js

```diff
@@ -0,0 +1,172 @@
+/* @flow */
+import React, { PureComponent } from "react";
+import { Trans } from "react-i18next";
+import { compose } from "redux";
+import { connect } from "react-redux";
+import { createStructuredSelector } from "reselect";
+import { TouchableOpacity, View, StyleSheet } from "react-native";
+import { findTokenById } from "@ledgerhq/live-common/lib/data/tokens";
+// $FlowFixMe
+import { NavigationScreenProp, SectionList } from "react-navigation";
+import i18next from "i18next";
+import SettingsRow from "../../../components/SettingsRow";
+import { showToken } from "../../../actions/settings";
+import { blacklistedTokenIdsSelector } from "../../../reducers/settings";
+import LText from "../../../components/LText";
+import CurrencyIcon from "../../../components/CurrencyIcon";
+import colors from "../../../colors";
+import { TrackScreen } from "../../../analytics";
+import HideEmptyTokenAccountsRow from "./HideEmptyTokenAccountsRow";
+import Close from "../../../icons/Close";
+
+type Props = {
+  blacklistedTokenIds: string[],
+  showToken: string => void,
+  navigation: NavigationScreenProp<*>,
+};
+
+const mapDispatchToProps = {
+  showToken,
+};
+
+const mapStateToProps = createStructuredSelector({
+  blacklistedTokenIds: blacklistedTokenIdsSelector,
+});
+
+const hitSlop = {
+  top: 16,
+  left: 16,
+  right: 16,
+  bottom: 16,
+};
+
+class AccountsSettings extends PureComponent<Props> {
+  static navigationOptions = {
+    title: i18next.t("settings.accounts.title"),
+  };
+
+  componentWillUnmount() {
+    // TODO trigger a refresh
```

Comment:
🤔

Reply:
Will be addressed once the sync rework is completed on https://github.com/LedgerHQ/ledger-live-mobile/pull/1208 and will look exactly like on desktop